src/components/SpecialResolution/CreateSpecialResolutionSummary.vue

```diff
@@ -0,0 +1,209 @@
+import { Action, Getter } from 'vuex-class'
+import {
+  ActionBindingIF,
+  ResourceIF,
+  FormIF,
+  CreateResolutionIF,
+  EntitySnapshotIF
+} from '@/interfaces/'
+import { CommonMixin, DateMixin } from '@/mixins/'
+import { HelpSection } from '@/components/common/'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker/'
+
+@Component({
+  components: {
+    HelpSection,
+    DatePickerShared
+  }
+})
+export default class CreateSpecialResolutionSummary extends Mixins(CommonMixin, DateMixin) {
+  @Getter getcreateResolution!: CreateResolutionIF;
+  @Getter getAppValidate!: boolean;
+  @Getter getSpecialResolutionConfirmValid!: boolean;
+  @Getter getEntitySnapshot!: EntitySnapshotIF
+
+  @Action setResolution!: ActionBindingIF;
+  @Action setSpecialResolutionConfirmStateValidity!: ActionBindingIF;
+
+  // Refs
+  $refs!: {
+    confirmResolutionChkFormRef: FormIF;
+  };
+
+  protected resolutionConfirmed = false;
+
+  /** Validation rule for checkbox. */
+  protected confirmCompletionResolution = [
+    v => {
+      return !!v
+    }
+  ];
+
+  protected async onResolutionConfirmedChange (resolutionConfirmed: boolean): Promise<void> {
+    // This is required as there are timing issues between this component and the CompleteResolutionSummary
+    // component.  The CompleteResolutionSummary isn't always able to detect that the confirm checkbox
+    // value has changed without using nextTick()
+    await this.$nextTick()
+    this.setResolution({
+      ...this.getcreateResolution,
+      resolutionConfirmed: resolutionConfirmed
+    })
+    this.setSpecialResolutionConfirmStateValidity(resolutionConfirmed)
+  }
+
+  /** The name section validity state (when prompted by app). */
+  get invalidSpecialResolutionConfirmSection (): boolean {
+    return this.getAppValidate && !this.getSpecialResolutionConfirmValid
+  }
+  /** The resolution date in readable. */
+  get resolutionDateText (): string {
+    return this.yyyyMmDdToPacificDate(this.getcreateResolution.resolutionDate, true)
+  }
+
+  /** The signing date in readable. */
+  get signingDate (): string {
+    return this.yyyyMmDdToPacificDate(this.getcreateResolution.signingDate, true)
+  }
+
+  /** The signing person details. */
+  get signingParty (): string {
+    if (this.getcreateResolution.signatory) {
+      const { givenName, additionalName = '', familyName } = this.getcreateResolution.signatory
+
+      return `${givenName} ${additionalName || ''} ${familyName}`
+    }
+    return ''
+  }
+
+  /** The company name. */
+  get companyName (): string {
+    // Hope this will be old company name even they change it
+    return this.getEntitySnapshot?.businessInfo?.legalName || ''
+  }
+
+  /** Set validate on file and pay click. */
+  @Watch('getAppValidate')
+  protected updateResolutionStepValidationDetail (): void {
+    !this.isJestRunning && this.$refs.confirmResolutionChkFormRef.validate()
```

Comment:
Add comment why this doesn't apply during unit tests.

Or could this work in unit tests with some mocking/stubbing?